DBpia.js

```diff
@@ -0,0 +1,162 @@
+{
+	"translatorID": "0c31f371-e012-4b1c-b793-f89ab1ae2610",
+	"label": "DBpia",
+	"creator": "Yunwoo Song, Philipp Zumstein",
+	"target": "^https?://[^/]+\\.dbpia\\.co\\.kr",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsi",
+	"lastUpdated": "2017-05-30 18:55:18"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	DBpia  Translator
+	Copyright © 2017 Yunwoo Song and Philipp Zumstein
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.indexOf('/Journal/ArticleDetail/')>-1) {
+		return "journalArticle";
+	} else if ((url.indexOf('/SearchResult/')>-1 || url.indexOf('/Journal/ArticleList/')>-1) && getSearchResults(doc, true)) {
+		return "multiple";
+	}
+}
+
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+	var rows = ZU.xpath(doc, '//div[@class="titleWarp"]/a');
+	for (var i=0; i<rows.length; i++) {
+		var href = rows[i].href;
+		var c = i + 1;
+		var title = ZU.trimInternal(rows[i].textContent);
+		if (!href || !title) continue;
+		if (checkOnly) return true;
+		found = true;
+		items[c] = title;
+	}
+	return found ? items : false;
+}
+
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "multiple") {
+		Zotero.selectItems(getSearchResults(doc, false), function (items) {
+			if (!items) {
+				return true;
+			}
+			var num = [];
+			for (var i in items) {
+				num.push(i);
+			}
+			for (var i=0; i<num.length; i++) {
+				var n = num[i];
+				var link = ZU.xpathText(doc, '//div[@class="subSearchResultList"]/div[' + n + ']/ul[1]/li[4]/dl/dt/div/a/@href')
+				if (link.indexOf('book')==-1) {
+					ZU.processDocuments(link, scrape);
+				} else {
+					var item = new Zotero.Item("book");
+					var title = ZU.xpathText(doc, '//div[@class="subSearchResultList"]/div[' + n + ']/ul[1]/li[4]/dl/dt/div/a');
+					item.title = title;
+					var creators = ZU.xpathText(doc, '//div[@class="subSearchResultList"]/div[' + n + ']/ul[1]/li[4]/dl/dd[1]');
+					if (creators) {
+						var creatorsList = creators.split(',');
+						for (var i=0; i<creatorsList.length; i++) {
+							item.creators.push(ZU.cleanAuthor(creatorsList[i], "author", true));
+						}
```

Comment:
If you mean "for this" by having the author field into a single field, I no longer care. When I tried saving an article with Zotero, full names of the authors were in "Last Name". This is good enough for me. With or without empty "first name" doesn't really matter. If, for some coding reasons, you prefer one way or another, you can change it, but I don't mind leaving it as is.